README.md

```diff
@@ -1,30 +1,90 @@
+* Simple management of Docker Apps across different teams and between different environments (Development/QA/Staging/Production)
+* Easy sharing of multi-service applications to container registries (e.g., [Docker Hub](https://hub.docker.com/) or [Docker Trusted Registry](https://docs.docker.com/ee/dtr/))
+* Having a clear separation of parameters to be modified at runtime 
+* Support for multiple orchestrators (Swarm or Kubernetes)
+* Provides the very same UX flow as the one for Docker images
+* Implements the [CNAB](https://cnab.io) industry standard
+* Offers full support of Docker Contexts
 
-Fundamentally, Compose files are not easy to share between concerns. Docker
-Application Packages aim to solve these problems and make Compose more useful
-for development _and_ production.
+## How does Docker App work?
 
-## Looking at an example
+The Docker App workflow is quite similar to the Docker image workflow. From an App definition, you can build an App image, share it on Docker Hub, and run your App on an orchestrator.
 
-Let's take the following Compose file. It launches an HTTP server which prints
-the specified text when hit on the configured port.
+![Image showing Docker CLI command flow](DockerAppCLIcommands.png)
+
+## Using Docker App
+
+Four primary steps comprise the Docker App process:
+1. Writing an App Definition
+1. Building an App Image
+1. Sharing the App on the Hub (optional)
+1. Running the App
+
+### Writing an App definition
+
+The first step in using Docker App is to write the App definition. This definition can be created (1) from an existing Compose file using the `docker app init` command (2) via  a template from the [Application Designer](https://docs.docker.com/ee/desktop/app-designer/), or (3) from scratch. 
+
+The App definition is a .dockerapp folder that contains three distinct pieces: metadata, a service list, and the parameters. 
+
+| File | Description |
+| :------------ | :------------ |
+| metadata.yml | metadata including the App name and version |
+| docker-compose.yml | Service list defined in a [Compose file](https://docs.docker.com/compose/compose-file/) |
+| parameters.yml | Parameters that can be changed when running the App |
+
+*Note: To store additional files in Docker Apps, such as* `prod.yml`*,* `test.yml` *or other config files, you need only to add these files to the *.dockerapp directory. All files will be packaged into the App image through the use of the* `docker app build`  *command.*
+
+### Building an App image
+
+Once the App definition is written, the next step is to build the App image from the App definition using the `docker app build` command. With this command you can tag your App image, set build-time variables, or make the build quiet. 
+Note that service images inside of an App image are immutable, meaning that the App version ties to a fixed list of service images (i.e., updating the images inside of a Docker App requires rebuilding the App image).
```

Comment:
Do we want to say why this is the case? i.e.: as this makes deploying applications more deterministic

Reply:
added